flake.nix

```diff
@@ -40,7 +40,6 @@
 
                     # ethereum
                     foundry-bin
-                    go-ethereum
```

Comment:
I suppose this means that the updated version of Geth isn't packaged for nix yet?

Reply:
Exactly, geth with shapella support required v11.x which is not packaged for nix, so we build from source code instead.

Reply:
I'm looking into packaging geth 11.0 and later for nix

Reply:
Cool. Though for this PR lets continue to build the source code manually.